pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/TestKitUtil.java

```diff
@@ -0,0 +1,479 @@
+package org.broadinstitute.dsm.juniperkits;
```

Comment:
Is this test utility specific to Juniper kits? IOW, can it be used to support non-Juniper kits?

Reply:
This was my first goal, but basically changing DSM so we can do that means changing around 20 classes and refactoring lots of different kit classes just to create a kit through the upload, which was not worth this ticket. So I am just using the method of Juniper Kit Creation just as a way to create the kit.